mobify/progressive-web-scaffold
# Split the cart data onto its own Redux branch

This completes (for now!) the architecture2.0 refactors to the Redux store. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1118
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Move the cart data from the `mini-cart` container to its own branch of the store.
- Remove things that were made redundant in the process (including the mini cart reducer)

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Make sure everything still works
